server-utils/src/suspendable_stream.rs

```diff
@@ -19,7 +17,7 @@ pub struct SuspendableStream<S> {
 	next_delay: Duration,
 	initial_delay: Duration,
 	max_delay: Duration,
-	timeout: Option<Delay>,
+	timeout: Option<Instant>,
```

Comment:
should maybe be renamed to `deadline`?

Reply:
Hm, `deadline` does indeed sound better but I'm having trouble understanding at a glance what this deadline is related to (it's until then that the "timeout" occurs, for lack of a better word).

Maybe something like `suspended_until` would be better in this case?

Reply:
Renamed in https://github.com/paritytech/jsonrpc/pull/628/commits/e265114ad3bdb875ff7bcae3e3c259e81b3d78e8